confz/confz_source.py

```diff
@@ -54,8 +54,9 @@ class ConfZEnvSource(ConfZSource):
     recommended, if ´allow_all´ is set."""
     remap: Optional[Dict[str, str]] = None
     """Certain environment variables can be mapped to config arguments with a different name."""
-
-
+    env_file: str = ".env"
```

Comment:
Should be a pathlib.Path and optional, with default value None

Reply:
Done.